core/src/main/resources/com/github/weisj/darklaf/task/darklaf_ja.properties

```diff
@@ -24,16 +24,16 @@
 Actions.cut                        = 切り取り
 Actions.copy                       = コピー
 Actions.paste                      = 貼り付け
-Actions.delete                     = 削除する
+Actions.delete                     = 削除
 Actions.close                      = 閉じる
 Actions.maximize                   = 最大化
 Actions.minimize                   = 最小化
-Actions.restore                    = 戻す
-Actions.closableTabbedPane.close   = 閉じる。Alt-クリックして他を閉じる
+Actions.restore                    = 元のサイズに戻す
+Actions.closableTabbedPane.close   = 閉じる。Alt キーを押しながらクリックすると、この夕ブ以外の他の夕ブを閉じます。
 Actions.save                       = 保存する
 Actions.revert                     = 元に戻す
 Actions.predefinedValues           = 事前定義された値
 
-Labels.colorChooser                = カラーチューザー
```

Comment:
Please squash those changes into the first commit :)
Same applies for the changes in `darklaf_zh` and `tabFrame_ja`